transport_nantes/mobilito/models.py

```diff
@@ -111,3 +111,50 @@ def save(self, *args, **kwargs) -> None:
             super().save(*args, **kwargs)
         except ValidationError as e:
             logger.error(f"Error while saving event : {e}")
+
+
+class InappropriateFlag(models.Model):
+    """Represent user spam/abuse reports on mobilito recording sessions.
+
+    We are publishing user-provided data to the internet.
+    Despite our efforts, it may happen that some sessions
+    are inappropriate, off brand, and/or shouldn't appear
+    publicly.  This mechanism lets visitors flag such inappropriate sessions
+    with an optional note to tell us why they believe the session is
+    inappropriate.
+
+    Notes on lingo :
```

Comment:
It's good that you comment these words.  Why don't you include the comments next to the definitions?

Reply:
The commit message for this model talks about the SessionReport model, which apparently does not exist.  Either the commit message or the commit is therefore wrong...